config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  
+   # Restful design routes
+  # resources :users, only: [:index, :show] do
+  #   resources :posts, only: [:index, :show]
```

Comment:
- [ ] Kindly make sure you remove the commented and unused code lines from your code base to make it more professional.

Reply:
remove all commented code as suggested by the reviewer